### DISCUSSION TITLE
Shift + Scroll Wheel should (but doesn't) perform a horizontal scroll

### DISCUSSION THREAD

**Discussion by anderson-arlen** [zed-industries/zed#12758] [Feature Requests]
This works in basically every other application I use and it's really frustrating that it doesn't work in Zed. What's the best way to enable this? Is there a config option somewhere?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
